docs/user/openstack/README.md

```diff
@@ -144,17 +146,19 @@ If you would like to set up an isolated development environment, you may use a b
 
 ## Running The Installer
 
+### Known Issues
+
+We are shipping the 4.2 release with a set of known issues. It is highly recommended that before running the installer, you check out the [known issues doccumentation](known-issues.md). We are trying to figure out workarounds for the time being, until we are able to resolve these bugs in the upcoming releases. To see the latest status of any bug, read through bugzilla or github link provided in that bug's description. If you know of a possible workaround that hasn't been documented yet, please comment in that bug's tracking link so we can address it as soon as possible. Also note that any bug listed in these documents is already a top priority issue for the dev team, and will be resolved as soon as possible. If you find more bugs during your runs, please read the section on [issue reporting](#reporting-issues) to learn how to file a bug report.
```

Comment:
We don't have installer-driven releases anymore, so I'd rather not have "the 4.2 release" here.  These docs describe whatever branch they happen to be on (in this case `master`, with `release-4.2` and `release-4.3` tracking), they don't need to explicitly name themselves.  How about:

> OpenStack support has [known issues](known-issues.md).  We are trying...

or similar?